packages/circuit-ui/components/NotificationToast/NotificationToast.docs.mdx

```diff
@@ -0,0 +1,137 @@
+import { Status, Props, Story } from '../../../../.storybook/components';
+import { NotificationToast } from '@sumup/circuit-ui';
+
+# NotificationToast
+
+<Status.Stable />
+
+<Story id="notification-notificationtoast--base" />
+<Props />
+
+The notification toast component non-disruptively provides quick, contextual or feedback based on the outcome of an action.
```

Comment:
This is not quite right: the toast can be triggered by an action (ex. saving a form), but it can also pop up without having been explicitly triggered (e.g. if we showed realtime sales in an app there could be a toast saying "You made a new sale").

This is actually (one of) the reason why toasts are so problematic—if they were only triggered by user interactions, then we could make them non-modal dialogs, send focus to them (this would allow actions on toasts), and send focus back to the element that triggered them on close.

(we may need to update this elsewhere in the docs)